src/test/java/uk/gov/digital/ho/hocs/casework/api/CaseDataCreateCaseIntegrationTest.java

```diff
@@ -70,7 +70,7 @@ public void shouldCreateACaseWithPermissionLevelOwner() throws JsonProcessingExc
         long numberOfCasesBefore = caseDataRepository.count();
 
         setupMockTeams("TEST", 5);
```

Comment:
Boyscout changes, we were passing in these params and not using it them, it looks like they've moved to setupMockTeams above and never been removed from this method params